src/test/unit/mocks/newSJPMocks.json

```diff
@@ -0,0 +1 @@
+{"document": {"publicationDate": "25th March 2022","version": "<O8|jZo>","documentName": "Uh"},"courtLists": [{"courtHouse": {"courtRoom": [{"session": [{"sittings": [{"hearing":[{"party":[{"partyRole":"PROSECUTOR","organisationDetails":{"organisationName":"TV Licensing","individualSurname":"test"}},{"partyRole":"ACCUSED","individualDetails":{"individualForenames":"Z","individualSurname":"ChambersFake","address":{"postCode":"BD17"}}}],"offence":[{"offenceTitle":"Use / install a television set without a licence"}]}]}, {"hearing":[{"party":[{"partyRole":"PROSECUTOR","organisationDetails":{"organisationName":"Driver and Vehicle Licensing Agency","individualSurname":"test"}},{"partyRole":"ACCUSED","individualDetails":{"individualForenames":"Z","individualSurname":"CollierFake","address":{"postCode":"BD17"}}}],"offence":[{"offenceTitle":"Keep a vehicle without a valid vehicle licence"}]}]}, {"hearing":[{"party":[{"partyRole":"PROSECUTOR","organisationDetails":{"organisationName":"Hampshire Police","individualSurname":"test"}},{"partyRole":"ACCUSED","individualDetails":{"individualForenames":"Z","individualSurname":"Edward-ColinsFake","address":{"postCode":"BD17"}}}],"offence":[{"offenceTitle":"Participate in gathering of two or more people in other outdoor place in Tier 4 area"}]}]}, {"hearing":[{"party":[{"partyRole":"PROSECUTOR","organisationDetails":{"organisationName":"Hampshire Police","individualSurname":"test"}},{"partyRole":"ACCUSED","individualDetails":{"individualForenames":"Z","individualSurname":"HortiguelaFake","address":{"postCode":"BD17"}}}],"offence":[{"offenceTitle":"Participate in gathering of more than two people in public outdoor place in Tier 4 area"}]}]}, {"hearing":[{"party":[{"partyRole":"PROSECUTOR","organisationDetails":{"organisationName":"TV Licensing","individualSurname":"test"}},{"partyRole":"ACCUSED","individualDetails":{"individualForenames":"Z","individualSurname":"KempFake","address":{"postCode":"BD17"}}}],"offence":[{"offenceTitle":"Use / install a television set without a licence"}]}]}, {"hearing":[{"party":[{"partyRole":"PROSECUTOR","organisationDetails":{"organisationName":"Driver and Vehicle Licensing Agency","individualSurname":"test"}},{"partyRole":"ACCUSED","individualDetails":{"individualForenames":"Z","individualSurname":"PlaschkeFake","address":{"postCode":"BD17"}}}],"offence":[{"offenceTitle":"Keep a vehicle without a valid vehicle licence"}]}]}, {"hearing":[{"party":[{"partyRole":"PROSECUTOR","organisationDetails":{"organisationName":"Transport for London","individualSurname":"test"}},{"partyRole":"ACCUSED","individualDetails":{"individualForenames":"Z","individualSurname":"StaceyFake","address":{"postCode":"BD17"}}}],"offence":[{"offenceTitle":"Enter a compulsory ticket area on the Transport for London regional railway network without a valid ticket"}]}]}]}]}]}}]}
```

Comment:
Only minor, however could we change the name of this file to be something more descriptive (list type would work)